source/app/service-providers/commands/rename-tag.ts

```diff
@@ -34,7 +34,7 @@ export default class RenameTag extends ZettlrCommand {
     const newName: string = arg.newName
 
     // First, retrieve all files from the FSAL
-    const allFiles = this._app.workspaces.getAllFiles()
+    const allFiles = (await this._app.fsal.getAllLoadedDescriptors())
```

Comment:
I think there are a few places where we could use this small helper function, but which I have forgotten. If you spot anywhere in this PR, feel free to ping.